user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md

```diff
@@ -108,7 +108,7 @@ There are also special types of `FieldDescriptors` that are purpose-made to stor
 
 ## CustomSectionDefinition properties
 
-The table below lists the properties of the `CustomSectionDefinition` object. (The base `SectionDefinition` object only exposes its ID.) The table also indicates whether a property can be used for filtering using the `SectionDefinitionExposers`.
+The table below lists the properties of the `CustomSectionDefinition` object. (The base `SectionDefinition` object only exposes its ID.) The table also indicates whether a property can be used for filtering using the `SectionDefinitionExposers`. Note that [the ITrackBase properties](xref:DOM_objects#itrackbase-properties) are also available on this type.
```

Comment:
@Skyline-ThomasGH Can I change this to:
"The *CustomSectionDefinition*  object also has the *ITrackBase* properties."?